woocommerce/woocommerce-ios
# Account creation: Separate email and password flow

Closes: #9407 


## Description

This PR optimizes the account creation flow by navigating to the login flow if the user inputs an email associated with an existing WPCom account:
- When the account creation screen is first presented, only the email field is present.
- When the submit button is tapped and the email is detected as already exists, the login flow with that email is presented.
- If the account creation request fails with an invalid password error, we show the password field for the user to enter a password. 

Also: updated wording on the account creation screen following the new design.

## Testing instructions

- On the prologue screen tap Get Started.
- Enter the email address of an existing email and tap Continue. Notice that the login flow is presented with the input email already populated.
- Navigate back to the account creation screen and change the email to a new one. Tap Continue and notice that the password field should now be available.

## Screenshots


https://user-images.githubusercontent.com/5533851/231069704-fea5e6ab-a081-40f6-974a-32b081eccb36.mp4



---
